test/contract.test.ts

```diff
@@ -165,12 +165,13 @@ describe('contract', function () {
   })
 
   it('should test contract (2)', async () => {
-    const testResult = await Greeter.tests.greet({ initialFields: { ...Greeter.defaultInitialFields, btcPrice: 1n } })
+    const initialFields = Greeter.getDefaultInitialFields()
```

Comment:
The naming is a bit misleading. How about `genInitialFieldsWithDefaultValues()`?